core/storage/exploration/gae_models.py

```diff
@@ -232,8 +232,9 @@ class ExplorationModel(base_models.VersionedModel):
         datastore_services.BooleanProperty(default=True, indexed=True))
     # A boolean indicating whether correctness feedback is enabled in this
     # exploration.
+    # DEPRECATED: Enable correctness feedback for all lessons.
```

Comment:
Move this to the bottom of the list of properties. See how we do it in 
core/storage/email/gae_models.py -- you should have a similar format and docstring here. Its value doesn't matter any more.

You don't need to make it default=True either since we shouldn't be reading it. When reading the storage model to the domain layer we should ignore the deprecated field entirely.

Reply:
Done